_locales/en/messages.json

```diff
@@ -2149,15 +2149,18 @@
     },
 
     "powerAmperageScale": {
-        "message": "Scale the output voltage to milliamps [1/10th mV/A]"
+        "message": "Scale"
     },
     "powerAmperageOffset": {
-        "message": "Offset in millivolt steps"
+        "message": "Offset in milliampere steps"
```

Comment:
Why has this been changed? It seems to be incorrect. The offset really is in millivolts (as measured by the ADC), not milliamperes. Here is the code from current.c
https://github.com/betaflight/betaflight/blob/master/src/main/sensors/current.c#L107-L112

And the scale units [1/10th mV/A] are also nice to display.

Reply:
@Cleric-K: Yep, that's correct, well spotted. Would you mind opening a pull request to fix this?